gtars/src/overlap/ailist.rs

```diff
@@ -66,11 +121,11 @@ where
             ends,
             max_ends,
             header_list,
-            data_list,
+            stored_intervals,
         }
     }
 
-    fn find(&self, start: I, end: I) -> Vec<Interval<I,T>> {
+    fn find(&self, start: I, end: I) -> Vec<Interval<I, T>> {
```

Comment:
Just an "engineering" thought. Not sure if it makes sense, should `find` just internally use `find_iter` that collects at the end before returning the result?